src/Microsoft.Extensions.Options/OptionsServiceCollectionExtensions.cs

```diff
@@ -69,5 +69,43 @@ public static IServiceCollection Configure<TOptions>(this IServiceCollection ser
 
         public static IServiceCollection ConfigureAll<TOptions>(this IServiceCollection services, Action<TOptions> configureOptions) where TOptions : class
             => services.Configure(name: null, configureOptions: configureOptions);
+
+        /// <summary>
+        /// Registers an action used to Initialize a particular type of options.
```

Comment:
Yeah I'll fix, global find/replace fail, I just cloned the ConfigureOptions files and replaced